sentry-spring-boot-starter/src/main/java/io/sentry/spring/boot/SentryRequestFilter.java

```diff
@@ -0,0 +1,36 @@
+package io.sentry.spring.boot;
+
+import com.jakewharton.nopen.annotation.Open;
+import io.sentry.core.IHub;
+import java.io.IOException;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+/** Pushes new {@link io.sentry.core.Scope} on each incoming HTTP request. */
+@Open
```

Comment:
why this class can't be `final`? just curious

Reply:
Spring creates proxies with CGLIB around beans and it's not possible to proxy final class with CGLIB.

Reply:
makes sense